apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/config/constants/RestOpenApiConstants.java

```diff
@@ -44,6 +44,7 @@ interface Parameters {
     String RAW = "raw";
     String TABLE_NAME = "tableName";
     String SORT = "sort";
+    String OPTIMIZE_MAP = "optimize_map";
```

Comment:
Beside naming choice, shouldn't this be capitalized as `optimizeMap` (see `tableName` above)?

Reply:
Fixed